.changeset/shy-needles-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': minor
```

Comment:
I waivered back and forth on minor/major. I don't think this package has breaking changes exactly. Once we update rafiki to this version the api implementation will be out of date but wont be broken. However, since our validator package uses the spec and checks for the new return type, I expect _that_ to break.

Reply:
Yeah this versioning stuff is tough. If we really want to follow semver, major makes sense since upgrading this package will cause requests from the client to break (given an older version of Rafiki). But on the other hand, nothing from the _user_/implementor of the client needs to change... 🤷‍♂️ 

Maybe this is fine for now then, in that case

Reply:
Yeah... to rephrase, the `open-payments` package doesn't break with this change. The `open-api` package, if used, breaks with this change. Which just highlights some weirdness of being tightly coupled via the spec. To be clear, I'm not suggesting there is a better way to do that, I just think it makes this weird. I'll keep as minor for now I guess.